scripts/test_workflow.sh

```diff
@@ -94,4 +94,10 @@ function test_workflow() {
 test_workflow
 check_ret $? "Test SQLFLow workflow failed"
 
+# test submit pai job using argo workflow mode
+if [ $SQLFLOW_submitter == "pai" ]; then
```

Comment:
This is not work in the current PR? Maybe we need to add `SQLFLOW_submitter=pai` in the `travis.yaml`

Reply:
`SQLFLOW_submitter=pai` tests now can only run in the internal environment